examples/flutter_gallery/lib/demo/material/date_and_time_picker_demo.dart

```diff
@@ -10,13 +10,9 @@ import 'package:intl/intl.dart';
 import '../../gallery/demo.dart';
 
 class _InputDropdown extends StatelessWidget {
```

Comment:
The original version wasn't formatted quite right and the new version is less right. Should be:

```
  const _InputDropdown({
    Key key,
    this.child,
    this.labelText,
    this.valueText,
    this.valueStyle,
    this.onPressed,
  }) : super(key: key);
```